### ISSUE TITLE
Dragging with compact editor doesn't follow the mouse

### ISSUE THREAD

**Issue by sfederici** [ScratchAddons/ScratchAddons#6549]
### Description

When dragging the first block after the "compact editor" addon is enabled the block is distant from the mouse pointer

### Steps to reproduce

1. enable the "compact editor" addon
2. click "create" in the top bar
3. drag the "move steps" block to the coding area
4. the block is at the right of the mouse pointer

### Demonstration(s) and additional context

<img width="1344" alt="Schermata 2023-08-18 alle 11 34 22" src="https://github.com/ScratchAddons/ScratchAddons/assets/2651868/0ac09feb-798d-4acf-9a76-17ab91072dd0">

### Environment

Browser: 116.0.2 (64 bit)
Scratch Addons version:  v1.34.0
Operating system: MacOS Big Sur

**Comment by WorldLanguages** [comment]
Weird, I think this is the first report we receive about this. Maybe this is only Firefox on MacOS or something like that?

**Comment by sfederici** [comment]
Could it be a wrong interaction between several addons? Here you are my ScratchAddons 
[scratch-addons-settings.json 2.zip](https://github.com/ScratchAddons/ScratchAddons/files/12381044/scratch-addons-settings.json.2.zip)
settings

**Comment by WorldLanguages** [comment]
Can't reproduce with the same settings on Chromium with Windows. Also tried Firefox with Windows.

@sfederici Check if you can get the bug to happen in Chromium with MacOS

**Comment by sfederici** [comment]
Can't reproduce in Chromium with MacOS

**Comment by WorldLanguages** [comment]
Maybe try disabling other extensions from your Firefox browser? Check if it happens in some other Mac device? (I don't have access to one at the moment)

**Comment by sfederici** [comment]
Is there a way to disable all extensions so that I can then enable only the "compact editor" extension?

**Comment by Samq64** [comment]
> Is there a way to disable all extensions so that I can then enable only the "compact editor" extension?

**Comment by sfederici** [comment]
No, sorry, I meant all addons, so that I can then enable only the "compact editor" addon

**Comment by WorldLanguages** [comment]
You could disable 15 addons at a time, reload, and see if the bug persists. If the bug is gone then it's one of those addons that caused the bug.

Then reimport your settings to restore them.

**Comment by Chiroyce1** [comment]
Firefox 116.0.3 (64-bit) on macOS Ventura

https://github.com/ScratchAddons/ScratchAddons/assets/97374054/f64ccfa9-c44c-49d5-8751-ee2e6acccf8f

I doubt this is a bug caused by Scratch Addons. It only *seems* to  happen with a tiny user base of Firefox on Mac, it still isn't reproducible, even with the same settings file.  @sfederici could you try going to `about:profiles`, creating a new profile and then trying out these exact reproduction steps with the same settings file? Sort of like a clean run with the same settings.

**Comment by sfederici** [comment]
> You could disable 15 addons at a time, reload, and see if the bug persists. If the bug is gone then it's one of those addons that caused the bug.
> 
> Then reimport your settings to restore them.

I found the interefering addon. It happens when the "Auto-hiding block palette" addon is enabled. It is immediately visible if I use the option "category click" for "toggle on...". Instead, if I use "palette are hover" it happens sometimes when I keep dragging the block back to the palette area (without leaving it in the script area)

**Comment by sfederici** [comment]
>  @sfederici could you try going to `about:profiles`, creating a new profile and then trying out these exact reproduction steps with the same settings file?

Created new profile, reinstalled ScratchAddons, loaded my settings file: same result. And I have to add something: when I leave the blocks in the script area, their color is the lighter one that is used when dragging them around with the "block tansparency" addon enabled (I use "Dragged transparency (%)" to 25%).

**Comment by WorldLanguages** [comment]
@sfederici So is this reproduceable just by enabling  "Auto-hiding block palette" with "category click" mode, and dragging a block right after clicking "create" on the navbar? (on Firefox, MacOS)

We can't really do much until someone else reproduces on Firefox MacOS. Hopefully Chiroyce1 will be able to.

**Comment by sfederici** [comment]
> @sfederici So is this reproduceable just by enabling "Auto-hiding block palette" with "category click" mode, and dragging a block right after clicking "create" on the navbar? (on Firefox, MacOS)

Yes. Of course even the "compact editor" addon must be enabled

**Comment by Chiroyce1** [comment]
I was able to reproduce this now, fresh install of SA and by enabling "Auto-hiding block palette" with "category click" mode with compact editor also enabled. Only happens the first time you select and drag a block, regardless of if you place it in the workspace or you leave it. Seems to be happening with all blocks as well.

**Comment by Samq64** [comment]
Could this be related to #5787?

**Comment by WorldLanguages** [comment]
> Could this be related to #5787?

I think that bug was fixed after a restart and only stayed during the browser session where the user experienced the update. As far as I know, this bug persists after a browser restart.
